### PR TITLE
Spelling_and_grammer_mistakes

### PR DESCRIPTION
**Pull request by Aniruddha120**
Speling and grammer mistakes are improved

### Summary

Spelling and grammer mistakes are improved

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -4,11 +4,11 @@
                            Version 2.0, January 2004
                         http://www.apache.org/licenses/
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
```
Comment (Aniruddha120) [inline LICENSE.txt:7]: It was not meant to make any changes in the license . Please ommit it.

Comment (atilag) [reply LICENSE.txt:7]: You have to undo the changes by yourself, we cannot just ignore this as the squash merge will treat everything as a block.

Comment (Aniruddha120) [reply LICENSE.txt]: I just removed the changes, will it work ??. as I have edited the many files altogether and I dont know how to revert changes for a single file. If possible please help.

Comment (atilag) [reply LICENSE.txt]: There seems to be other changes affecting this file, remove them too and we will be good to go :)

**Comment by Aniruddha120** [comment]
I think this is alright now, please check it..

**Comment by Aniruddha120** [comment]
> 
> 
> Thanks!

You are most welcome!

**Pull request merged**